snapshot/generate_test.go

```diff
@@ -64,27 +60,9 @@ func genSmartContractAccount(nonce uint64, balance *big.Int, storageRoot common.
 	})
 }
 
-// TODO-Klaytn: To enable logging in the test code, we can use the following function.
-// This function will be moved to somewhere utility functions are located.
-func enableLog() {
-	usecolor := term.IsTty(os.Stderr.Fd()) && os.Getenv("TERM") != "dumb"
-	output := io.Writer(os.Stderr)
-	if usecolor {
-		output = colorable.NewColorableStderr()
-	}
-	glogger := log.NewGlogHandler(log.StreamHandler(output, log.TerminalFormat(usecolor)))
-	log.PrintOrigins(true)
-	log.ChangeGlobalLogLevel(glogger, log.Lvl(5))
-	glogger.Vmodule("")
-	glogger.BacktraceAt("")
-	log.Root().SetHandler(glogger)
-}
-
 // Tests that snapshot generation from an empty database.
 func TestGeneration(t *testing.T) {
-	if false {
-		enableLog()
-	}
+	log.EnableLogForTest(log.LvlCrit, log.LvlTrace)
```

Comment:
In this kind of tests, it seems that `enableLog()` is not called unless we change the test code.. however, it seems you changed the code to call `enableLog()`. Is this intended?

Reply:
Yes you understood correctly. It is intended based on an assumption.

I guessed original author of the test wanted to do something like `if testing.Verbose() { enableLog() }`. That should be the purpose of the if statement.

Therefore I've replaced it with the equivalent that does not require manual code editing. I think adding `-v` flag is more convenient because it is quicker and creates zero git diff.

Reply:
Oh i see, it's definitely better than before..